packages/tsconfig.base.json

```diff
@@ -14,6 +14,9 @@
     "baseUrl": "./",
     "lib": ["es6", "es7"],
     "strict": false,
+    "paths": {
```

Comment:
will be merged anyway

Reply:
yup, but there were some problems with recognition from tests point of view as well, it's good to have clear instruction where to search for it for other packages :)